test/dsl_test.go

```diff
@@ -520,8 +520,8 @@ func lsdir(name string, contents m) fileOp {
 	}, Defaults}
 }
 
-func (c *ctx) getNode(filepath string, create bool, isFile bool) (
-	Node, bool, error) {
+func (c *ctx) getNode(filepath string, create bool, isFile bool,
```

Comment:
so many bools...can you make types for each of them with consts so all the callers are clear?